roles/ci_network/tasks/add-static-leases.yml

```diff
@@ -0,0 +1,57 @@
+---
+- name: Add DHCP reservations
+  when:
+    - cifmw_network_dnsmasq_config is defined
+    - "'dhcp' in cifmw_network_dnsmasq_config"
+    - "'network' in cifmw_network_dnsmasq_config['dhcp']"
```

Comment:
why the double quote here?

Reply:
because of the `when foo in bar` - ansible `when` is slightly stupid on that. Unless it changed?